### PR TITLE
[VEN-2373]: Add extra documentation related to XVS Bridge

### PR DESCRIPTION
**Pull request by GitGuru7**

### CODE REVIEW

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 1]
```diff
@@ -140,6 +140,10 @@ In the event that a `Bridge` contract needs replacement, such as due to a securi
 
 These steps ensure a secure and systematic replacement of a `Bridge` contract, maintaining the integrity of the token. Simultaneously, on the BNB chain, the locked XVS will be transferred and locked in the other `Bridge` contract, ensuring a fix total supply of XVS.
 
+### 6.10. Default Downtime
+- Currently, the `Bridge` relies on a single relayer, the [default](https://layerzero.gitbook.io/docs/technical-reference/mainnet/default-config) by LayerZero, to generate proofs and submit them to target chains. While this configuration is functional, it's important to be aware of the potential implications. If the relayer goes offline or encounters problems, there's no immediate backup to maintain bridge functionality, potentially delaying or preventing transactions. In the event of unforeseen downtime affecting the default LayerZero relayer, we will designate an authorized user who can temporarily generate proofs and submit them on the target network on behalf of the relayer. This authorization is granted only in exceptional circumstances via VIP.
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
- Currently, the `Bridge` relies on a single relayer, the [default](https://layerzero.gitbook.io/docs/technical-reference/mainnet/default-config) by LayerZero, to generate proofs and submit them to target chains. While this configuration is functional, it's important to be aware of the potential implications. If the relayer goes offline or encounters problems, there's no immediate backup to maintain bridge functionality, potentially delaying or preventing transactions. In the event of unforeseen downtime affecting the default LayerZero relayer, a wallet can be authorized to temporarily generate proofs and submit them on the target network on behalf of the relayer. This authorization is granted only in exceptional circumstances via VIP.
```

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 2]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
+
+In the event of a failed transaction, follow a step-by-step process using block explorers and the `retryMessage` function to retry transactions on the respective blockchain. Here's a detailed guide:
+
+1. **Identify the Failed Transaction:**
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md:224]: I would suggest using LayerZeroScan to identify the transaction in the target network, providing the transaction hash in the source network. The `MessageFailed` event is in the transaction sent to the target network

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 3]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+   - Use a block explorer (e.g., Etherscan) to locate the failed transaction on the respective blockchain.
+
+2. **Examine the FailedMessage Log:**
+   - Access the emitted events of the failed transaction and specifically examine the `failedMessage` log. This log contains essential function parameters needed for the retry.
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]: I think the name of the event is `MessageFailed`, not `failedMessage`. For example: https://etherscan.io/tx/0x2f3dd0eb843a4e27af1e1ebafc14d37766a27de07c5383e7f7e2e5f2e754b5b2#eventlog

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 4]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+
+3. **Extract Function Parameters:**
+   - From the `failedMessage` log, extract the following essential function parameters:
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
   - From the `MessageFailed` log, extract the following essential function parameters:
```

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 5]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+     - `_srcChainId`
+     - `_srcAddress`
+     - `_nonce`
+     - `_payload`
+
+4. **Construct a RetryMessage:**
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md:237]: Mention the contract name where the `retryMessage` function is. Mention the network where this has to be done

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 6]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md:220]: Generally, I would add:

* Mention and link (to the `main` branch version) to the contract where the retryMessage function is available
* Mention and link (to the `main` branch version) to the contract where the `MessageFailed` event is emitted

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 7]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+
+3. **Extract Function Parameters:**
+   - From the `MessageFailed` log, extract the following essential function parameters:
+     - `_srcChainId`
+     - `_srcAddress`
+     - `_nonce`
+     - `_payload`
+
+4. **Construct a RetryMessage:**
+   - In the event of a transaction failure on the BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTSrc` contract on the BNB chain. Use the parameters extracted from the `MessageFailed` log for this operation. Conversely, if the transaction fails on any network other than BNB, invoke the `retryMessage` function of the `XVSProxyOFTDest` contract in that network.
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
   - In the event of a transaction failure on the BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTSrc` contract on the BNB chain. Use the parameters extracted from the `MessageFailed` log for this operation. Conversely, if the transaction fails on any network other than BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTDest` contract in that network. 
```

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 8]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
+
+In the event of a failed transaction, follow a step-by-step process using block explorers and the [`retryMessage`](https://github.com/VenusProtocol/token-bridge/blob/main/contracts/Bridge/BaseXVSProxyOFT.sol#L368) function to retry transactions on the respective blockchain. Here's a detailed guide:
+
+1. **Identify the Failed Transaction:**
+   -  Use [LayerZero scan](https://layerzeroscan.com) to identify the failed transaction within the target network by providing the transaction hash from the source network where the transaction was initiated.
```
Comment (chechu) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
   - Use [LayerZero scan](https://layerzeroscan.com) to identify the failed transaction within the target network by providing the transaction hash from the source network where the transaction was initiated.
```

Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [102c36f](https://github.com/VenusProtocol/venus-protocol-documentation/pull/156/commits/102c36f81b96a3d7d50b893c1c0bbce640bb1ab0)

File: technical-reference/reference-technical-articles/xvs-bridge.md (hunk 7 above)
Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [102c36f](https://github.com/VenusProtocol/venus-protocol-documentation/pull/156/commits/102c36f81b96a3d7d50b893c1c0bbce640bb1ab0)

**Review by chechu — APPROVED** [review]

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 9]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
 - The transfer amount exceeds the single or daily receiving transaction limit.
 - The recipient is blacklisted by the XVS token.
 - The minting cap on the destination bridge is exceeded.
+
+### Retry Mechanism for Failed Transactions
+
+In the event of a failed transaction, follow a step-by-step process using block explorers and the [`retryMessage`](https://github.com/VenusProtocol/token-bridge/blob/main/contracts/Bridge/BaseXVSProxyOFT.sol#L368) function to retry transactions on the respective blockchain. Here's a detailed guide:
```
Comment (coreyar) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
In the event of a failed transaction, follow the bellow step-by-step process using block explorers and the [`retryMessage`](https://github.com/VenusProtocol/token-bridge/blob/main/contracts/Bridge/BaseXVSProxyOFT.sol#L368) function to retry transactions on the respective blockchain. Here's a detailed guide:
```

File: technical-reference/reference-technical-articles/xvs-bridge.md [hunk 10]
```diff
@@ -212,3 +216,23 @@ In addition to the core functionality, the XVS Cross-chain Bridge includes addit
...
+
+3. **Extract Function Parameters:**
+   - From the `MessageFailed` log, extract the following essential function parameters:
+     - `_srcChainId`
+     - `_srcAddress`
+     - `_nonce`
+     - `_payload`
+
+4. **Construct a RetryMessage:**
+   - In the event of a transaction failure on the BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTSrc` contract on the BNB chain. Use the parameters extracted from the `MessageFailed` log for this operation. Conversely, if the transaction fails on any network other than BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTDest` contract in that network.
```
Comment (coreyar) [inline technical-reference/reference-technical-articles/xvs-bridge.md]:
```suggestion
   - In the event of a transaction failure on the BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTSrc` contract on the BNB chain. Use the parameters extracted from the `MessageFailed` log for this operation. Conversely, if the transaction fails on any network other than BNB chain, invoke the `retryMessage` function of the `XVSProxyOFTDest` contract on that network. 
```

File: technical-reference/reference-technical-articles/xvs-bridge.md (hunk 9 above)
Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [e6ab5bc](https://github.com/VenusProtocol/venus-protocol-documentation/pull/156/commits/e6ab5bc02d01c02eb4cf937209216b949549eba1)

File: technical-reference/reference-technical-articles/xvs-bridge.md (hunk 10 above)
Comment (GitGuru7) [reply technical-reference/reference-technical-articles/xvs-bridge.md]: [e6ab5bc](https://github.com/VenusProtocol/venus-protocol-documentation/pull/156/commits/e6ab5bc02d01c02eb4cf937209216b949549eba1)

**Review by coreyar — APPROVED** [review]

**Pull request merged**